artemis-chart/templates/backend-deployment.yaml

```diff
@@ -26,7 +28,7 @@ spec:
         - mountPath: /pvc
           name: backend-pvc
           subPath: configs
-        command: ['sh', '-c', 'cp -n -u /configmaps/config.yaml /configmaps/logging.yaml /configmaps/services.conf /pvc/']
+        command: ['sh', '-c', 'false | cp -i /configmaps/config.yaml /configmaps/logging.yaml /configmaps/services.conf /pvc/ 2> /dev/null']
```

Comment:
What is this false about?

Reply:
it asks if you want to replace the files and it answer no :P